cpp/src/arrow/python/gdb.cc

```diff
@@ -29,6 +29,7 @@
 #include "arrow/scalar.h"
 #include "arrow/table.h"
 #include "arrow/type.h"
+#include "arrow/testing/gtest_util.h"
```

Comment:
I'm not sure we want this linked to googletest, we should stick to the old version.